packages/component-library/src/components/form/mt-datepicker/mt-datepicker.vue

```diff
@@ -58,7 +58,9 @@
       </template>
     </vue-datepicker>
 
-    <template v-if="isTimeHintVisible">
+    <mt-field-error v-if="error" :error="error" />
+
+    <template v-else-if="isTimeHintVisible">
```

Comment:
The time hint could be visible. No matter field has error or not.

```suggestion
    <template v-if="isTimeHintVisible">
```


Reference by `mt-url-field`


<img width="1009" alt="image" src="https://github.com/user-attachments/assets/b7ec27b8-228e-4022-b752-f347fe3829c4" />